packages/lib-classifier/src/plugins/tasks/SimpleDropdownTask/models/SimpleDropdownAnnotation.js

```diff
@@ -0,0 +1,18 @@
+import { types } from 'mobx-state-tree'
+import Annotation from '../../models/Annotation'
+
+const SimpleDropdown = types.model('SimpleDropdown', {
+  value: types.optional(types.frozen({
+    value: types.string,  // Value selected by the user
+    option: types.boolean,  // i.e. is this value in the list of preset options, or is it a custom "other" value from the user?
+  }), null),
+})
+  .views(self => ({
+    get isComplete () {
+      return !!(self.value && self.value.value && self.value.value.length > 0)
```

Comment:
[Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) is stage 4 and is fully supported now, so this can be a bit cleaner now:
`self.value?.value?.length` though I still recommend not using `value.value`

Also the greater than operator should evaluate to a boolean, so I'm not clear why casting to a boolean again using `!!` is necessary.

Reply:
The `!!` was necessary because if `self.value` was undefined, `isComplete()` would evaluate to `undefined` and I'd get an error saying "isComplete was expected to be bool, but instead was undefined"

I'll check if those ?. optional chaining works around this